modules/gwlb/versions.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 1.0, < 2.0"
+  required_version = ">= 1.3, < 2.0"
```

Comment:
please bump tf trequirements to min = 1.5 and provider to min 3.80

Reply:
done